autoware_lanelet2_extension/docs/extra_lanelet_subtypes.md

```diff
@@ -43,3 +43,29 @@ Sample pedestrian lane in .osm format is shown below:
     <tag k="one_way" v="yes"/>
   </relation>
 ```
+
+## Bicycle Lane (format_version >= 2)
+
+Although `bicycle_lane` subtype is already defined in the original [Lanelet2 documentation](https://github.com/fzi-forschungszentrum-informatik/Lanelet2/blob/master/lanelet2_core/doc/LaneletAndAreaTagging.md#subtype-and-location), Autoware compatible with `format_version>=2` supports `bicycle_lane` at component level.
+
+- refers: `road` lanelet adjacent to the objective `bicycle_lane` should share the same boundary LineString
+- specification:
+  - A `bicycle_lane` is connected to/from only `bicycle_lane` lanelets and is adjacent to a `road` lanelet. Namely in Left(Right) hand side traffic rules, it is on the left(right) side of `road` lanelet.
+- Although pullover on `bicycle_lane` is normally prohibited by the law, it is allowed to place a `road_shoulder` adjacent to a `bicycle_lane`. However the autoware implementation may not fully support appropriate behavior in such cases.
```

Comment:
It's slightly abrupt to start discussing about pullover behavior in this line.

My suggestion is to maybe explain the expected usecase of bicycle_lane after/before the first paragraph of Bicycle Lane section

e.g.,
In Autoware, bicycle lanes are used to identify paths dedicated to cyclists. These lanes can assist in predicting the future paths of bicycles and indicate lanes that Autoware vehicles should avoid or keep clear of.